packages/generator/generators/draft/templates/README.txt

```diff
@@ -0,0 +1,27 @@
+---
+title: "<%= componentName %>"
+navTitle: "<%= componentName %>"
+summaryParagraph: <%= componentName %>s are a neat component
+tags: [""]
+demoStoryId:  <%= componentName %>-react--default-kaizen-site-demo
+---
+
```

Comment:
Is it worth including DOs and DONTs imports for copy guidelines even without adding default sections for copy guidelines? Just so people can use them without needing to know about the imports? 

```
import DoAndDontContainer from "docs-components/DoAndDontContainer"
import Do from "docs-components/Do"
import Dont from "docs-components/Dont"
```